src/meetup.py

```diff
@@ -0,0 +1,116 @@
+# this class is about to collect all meetups from einundzwanzig.space/meetups.json
+# and returns a (Telegram-)message with all meetups (command: /meetup)
+
+import config
+
+from telegram.ext.callbackcontext import CallbackContext
+from telegram.update import Update
+
+import urllib.request, json 
+
+class Meetup:
+
+    def __init__(self, name: str, region: str, url: str) -> None:
+        self.name = name
+        self.region = region
+        self.url = url
+
+
+# variables
+# get json file from website
+url = urllib.request.urlopen("https://einundzwanzig.space/meetups.json")
+json_string = json.loads(url.read().decode())
+
+# save json file as string
+json_string = str(json_string)
+
+#count meetups
+#use "{" to count, for each "{" there is one meetup
+count = 0
+
+for i in str(json_string):

```

Comment:
Why convert to a string and then do manual json processing? I think it would be much easier (and more readable) if you use the parsed dict and then use a loop to go over every meetup. See soundboard code for examples.